signinapp/templates/search/base.html.jinja2

```diff
@@ -1,17 +1,21 @@
 {% extends "base.html.jinja2" %}
 {% from 'bootstrap5/form.html' import render_form, render_messages %}
-{% block title %}Search - Chop Shop Sign In{% endblock %}
+{% block title %}
+  Search - Chop Shop Sign In
+{% endblock title %}
 {% block content %}
   <div id="searchContainer" class="container pt-3">
     <div class="row">
       {{ render_messages() }}
       <div class="col">{{ render_form(form, extra_classes="search-form") }}</div>
       <div class="col-lg-6 py-3">
-        <table class="table table-striped table-hover table-dark">
-          {%- block searchtable -%}
-          {%- endblock -%}
-        </table>
+        <div class="table-responsive">
+          <table>
```

Comment:
Can we put this class on the table itself so we don't need the extra element or does that break stuff?

Reply:
according to the bootstrap docs, the table has to be wrapped with `table-responsive` in order for it to work. The `table-responsive` class allows for phone users and smaller screens to scroll tables.